spec/rails_helper.rb

```diff
@@ -61,4 +61,20 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+  config.include Capybara::DSL
```

Comment:
Curious what error you were running into with Capybara that required you to add this!